lib/std/fs/test.zig

```diff
@@ -13,6 +13,11 @@ const File = std.fs.File;
 const tmpDir = testing.tmpDir;
 const tmpIterableDir = testing.tmpIterableDir;
 
+// ensure tests for fs/wasi.zig are run
+comptime {
+    _ = std.fs.wasi;
+}
```

Comment:
This should be a `test` block instead of `comptime` so that it only affects when `zig test` is used. And it should check the target is actually wasi.

```zig
test {
    if (builtin.os.tag == .wasi) {
        _ = std.fs.wasi;
    }
}
```